WordPress/src/main/java/org/wordpress/android/ui/stories/prefs/StoriesPrefs.kt

```diff
@@ -0,0 +1,128 @@
+package org.wordpress.android.ui.stories.prefs
+
+import android.content.Context
+import android.net.Uri
+import androidx.preference.PreferenceManager
+import com.wordpress.stories.compose.story.StoryFrameItem
+import com.wordpress.stories.compose.story.StoryFrameItem.BackgroundSource.FileBackgroundSource
+import com.wordpress.stories.compose.story.StoryFrameItem.BackgroundSource.UriBackgroundSource
+import com.wordpress.stories.compose.story.StorySerializerUtils
+
+object StoriesPrefs {
```

Comment:
Hi @mzorz any specific reason why this class was made a Singleton since we can utilize Dagger to create a `@Singleton` scope and we could make it testable? LMK 🙏

Reply:
the reason is: I keep forgetting, thanks a ton for reminding me an being on top of this, covering our back :D 
Made the needed changes in f6d92d293c

Reply:
More than happy to help! 🙇